plugins/generator-1.18.2/forge-1.18.2/templates/modbase/accesstransformer.cfg.ftl

```diff
@@ -1,13 +1,24 @@
 <#if w.hasToolsOfType("Fishing rod")>
 public net.minecraft.world.entity.projectile.FishingHook m_37136_(Lnet/minecraft/world/entity/player/Player;)Z #shouldStopFishing
 </#if>
+
 <#if w.hasElementsOfType("dimension")>
 public net.minecraft.world.level.levelgen.carver.WorldCarver f_64983_ #replaceableBlocks
 public net.minecraft.client.renderer.DimensionSpecialEffects f_108857_ #EFFECTS
 </#if>
+
+<#if w.hasElementsOfType("biome")>
+public net.minecraft.world.level.biome.MultiNoiseBiomeSource <init>(Lnet/minecraft/world/level/biome/Climate$ParameterList;Ljava/util/Optional;)V #constructor
+public-f net.minecraft.world.level.biome.MultiNoiseBiomeSource f_48438_ #preset
+public-f net.minecraft.world.level.biome.MultiNoiseBiomeSource f_48435_ #parameters
+public-f net.minecraft.world.level.chunk.ChunkGenerator f_62137_ #biomeSource
+public-f net.minecraft.world.level.chunk.ChunkGenerator f_62138_ #runtimeBiomeSource
+</#if>
+
 <#if w.hasGameRulesOfType("Number")>
 public net.minecraft.world.level.GameRules$IntegerValue m_46312_(I)Lnet/minecraft/world/level/GameRules$Type; #create
 </#if>
+

```

Comment:
Number and logic game rules are the same mod element type, no need to split them I think.

Reply:
They are split for reason one can only make number and not logic ..

Reply:
@KlemenDEV I mean the empty line in-between.